File: hard/1_torunament_winner/tournament_winner_test.py
import unittest
from tournament_winner import tournamentWinner

class TestTournamentWinner(unittest.TestCase):
    def test_tournament_winner(self):
        competitions = [
            ["HTML", "C#"],
            ["C#", "Python"],
            ["Python", "HTML"],
        ]
        results = [0, 0, 1]
        expected = "Python"
        actual = tournamentWinner(competitions, results)
        self.assertEqual(expected, actual)

File: hard/1_torunament_winner/tournament_winner_test.go
package main

import (
	"github.com/stretchr/testify/require"
	"testing"
)

func TestTournamentWinner(t *testing.T) {
	competitions := [][]string{
		{"HTML", "C#"},
		{"C#", "Python"},
		{"Python", "HTML"},
	}
	results := []int{0, 0, 1}
	expected := "Python"
	actual := TournamentWinner(competitions, results)
	require.Equal(t, expected, actual)
}
